usr/lib/cca_stdll/cca_specific.c

```diff
@@ -312,6 +312,11 @@ struct cca_private_data {
         unsigned int rel;
         unsigned int mod;
     } version;
+    struct {
+        unsigned int ver;
+        unsigned int rel;
+        unsigned int mod;
+    } min_card_version;
```

Comment:
Since we have the second same version struct again here, I would suggest to add a struct definitions just in front of the struct cca_private_data like this:

```
struct cca_version {
        unsigned int ver;	
        unsigned int rel;	
        unsigned int mod;
    };	  
``` 

and use the following inside cca_private_data  (and everywhere else where the version struct is used:
```
struct cca_version cca_lib_version;
struct cca_version min_card_version;
```

While at it, please rename the 'version' field to 'cca_lib_version' or similar, indicating who's version it holds.

Reply:
Done.